integrations/langchain4j/providers/ollama/src/main/java/module-info.java

```diff
@@ -14,19 +14,17 @@
  * limitations under the License.
  */
 
-import io.helidon.common.features.api.Feature;
+import io.helidon.common.features.api.Features;
 import io.helidon.common.features.api.HelidonFlavor;
-import io.helidon.common.features.api.Preview;
 
 /**
  * Provides integration with Ollama models.
  */
-@Feature(value = "Langchain4j Ollama",
-         description = "Langchain4j Ollama Provider Integration",
-         in = {HelidonFlavor.SE, HelidonFlavor.MP},
-         path = {"Langchain4j", "Ollama"}
-)
-@Preview
+@Features.Name("Langchain4j Ollama")
```

Comment:
Please check all modules - if using new Feature API, you must use `codegen` instead of `processor`